hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -23,24 +24,40 @@
 
   // Import additional common sim cfg files.
   // TODO: remove imported cfgs that do not apply.
-  import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+  import_cfgs: [
+      // Project wide common sim cfg file
+      "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+      // Config file to pull in the correct flags for otbn_memutil
+      "{proj_root}/hw/ip/otbn/dv/memutil/otbn_memutil_sim_opts.hjson"
+  ]
+
+  // Add options needed to compile against otbn_memutil
+  en_build_modes: ["{tool}_otbn_memutil_build_opts"]
 
   // Add additional tops for simulation.
   sim_tops: ["-top {name}_bind"]
 
   // Default iterations for all tests - each test entry can override this.
-  reseed: 50
+  reseed: 1
 
   // Default UVM test and seq class name.
   uvm_test: otbn_base_test
   uvm_test_seq: otbn_base_vseq
 
+  // The directory where ELF binaries should be found.
+  //
+  // TODO: This needs deciding upon. Eventually, it might be nice to allow this
+  //       to be overridden on the command line ("I've got a bunch of ELF files
+  //       that stress my new feature; please run the test suite with them")
+  otbn_elf_dir: "{sw_build_dir}"
```

Comment:
What populates this directory with the elf files? If that is expected to be done externally, then we cannot run OTBN DV tests in private CI or the nightly regressions yet.

Reply:
That's true! Hence the discussion I started on slack about how to make this work. I'm pretty certain this code will change once we've figured out how to do it: this is just a stand-in so that I can make some progress while we work that out.

Reply:
I will think about this today and try to come up with a solution.

Reply:
Brill, thanks.